pyccl/tracers.py

```diff
@@ -1,9 +1,68 @@
 from . import ccllib as lib
 from .core import check
-from .background import comoving_radial_distance, growth_rate, growth_factor
+from .background import comoving_radial_distance, growth_rate, growth_factor, Sig_MG
 from .pyutils import _check_array_params, NoneArr
 import numpy as np
 
+def get_MG_transfer_function(cosmo, z, k_MG=None):
+    """ This function allows to obtain the function Sigma(z,k) (1 or 2 D vector)
+        for an array redshift (i.e. scale factor) coming from the redshift 
+        distributiorn (defined by the user) and a single value or an array of k 
+        specified by the user. We obtain then Sigma(z,k) as a 1D array for those z 
+        and k arrays and then convert it to a 2D array taking into consideration 
+        the given sizes of the arrays for a and k. The MG parameter array goes then
+        as a multiplicative factor within the MG transfer function. 
+        If k is not specified then only a 1D array for Sigma(a,k=0) is used.
+
+    Args:
+        cosmo (:class:`~pyccl.core.Cosmology`): cosmology object used to
+            transform redshifts into distances.
+        z (float or tuple of arrays): a single z value (for z_CMB for example) 
+            or a tuple of arrays (z, N(z)) giving the redshift distribution of the
+            objects. The units are arbitrary; N(z) will be normalized to unity.
+	k_MG (float or array): a single k value or an array of k for which we 
+            calculate the MG parameter Sigma(a,k). For now, the k range should 
+            be limited to linear scales. 
+    """
+	
+    if isinstance(z, float):
+        a_CMB=1/(1+z)
```

Comment:
I would prefer to keep this naming more general (e.g. a_single), although I understand the CMB is the most probable use here.

Reply:
That will work. Done!